events/ome-community-meeting-2020/day2/index.html

```diff
@@ -10,7 +10,8 @@
       vid_title1: Ontology-based imaging data integration using OME-OWL
       vid_speaker1: "Norio Kobayashi"
       vid_aff1: RIKEN
-      vid_link1: "../../../img/icons/placeholder_video.svg"
+      vid_link1: "https://downloads.openmicroscopy.org/presentations/2020/Dundee/main-talks/Norio_Kobayashi.mp4"
+      vid_poster3: "/img/video/norio_kobayashi.png"
```

Comment:
Is this file supposed to be added as part of this PR?

Reply:
Yes, of course, thank you @sbesson